docs/modules/ROOT/pages/cli.adoc

```diff
@@ -287,6 +288,9 @@ Scaffold a simple Nile project.
 
 Run a local {starknet-devnet} node.
 
+NOTE: Accepts all the options that the starknet-devnet node accepts by default. Check the {devnet-doc} for a full
+list of them.
+
```

Comment:
I added it, removed it, and added it again a couple of times thinking about the design. I'm not too comfortable 100 percent with either of the versions, because I wanted to let it as clear as possible for the user that he can use these arguments from starknet-devnet, but not adding all of them here. At the same time, I feel the user can check the command doc in a quick glance and miss this "Starknet Devnet Options".

Reply:
I'm ok with removing the note, but I still have the feeling the user can miss this.

Reply:
Haha yeah, I see the dilemma. If we remove `host` and `port`, there's much less noise around the `node` section. I think it's enough, no? Here's a crude example of how I'm seeing it:

### node
```
----
nile node
----
Run a local [starknet-devnet](https://github.com/Shard-Labs/starknet-devnet/) node.

Starknet Devnet Options
This command exposes...
```

Reply:
Thanks. I think that should be enough indeed.